libs/community/tests/unit_tests/llms/test_imports.py

```diff
@@ -92,11 +97,17 @@
     "VolcEngineMaasLLM",
     "WatsonxLLM",
     "SparkLLM",
+    "ChatDatabricks",
 ]
 
 
 def test_all_imports() -> None:
     """Simple test to make sure all things can be imported."""
+    from langchain_core.language_models import BaseChatModel
+
     for cls in llms.__all__:
-        assert issubclass(getattr(llms, cls), BaseLLM)
+        if cls in ["ChatDatabricks", "ChatMlflow"]:
```

Comment:
Hey @leo-gan ! I believe the old test was correct (confirm we're only exporting BaseLLM subclasses from `.llms`) - why are you trying to export chat models from `.llms`?

Reply:
Those two models were not presented in ___all___ . But they are presented in _import_databricks_chat() and _import_mlflow_chat(). They are exposed with get_type_to_cls_dict(). 
So, I have an option to remove _import_databricks_chat and _import_mlflow_chat OR add both classes to __all___. Otherwise, the access symmetry is broken.
Maybe there is a reason for this asymmetry, but if this asymmetry is not commented/documented, I can assume this asymmetry is not intentional.
Those added classes are not subclasses of `BaseLLM` but `BaseChatLLM`.

Reply:
The asymmetry is a middle ground to correct a past mistake by deprecating those, but didn't want to make them *more* accessible / advertise they're exported here.

Will remove the changes that don't make sense and merge